hadoop-ozone/dist/src/main/smoketest/basic/ozone-shell.robot

```diff
@@ -121,7 +121,7 @@ Test key handling
     ${result} =     Execute             ozone sh key info ${protocol}${server}/${volume}/bb1/key1 | jq -r '. | select(.name=="key1")'
                     Should contain      ${result}       creationTime
     ${result} =     Execute             ozone debug chunkinfo ${protocol}${server}/${volume}/bb1/key1 | jq -r '.[]'
-                    Should contain      ${result}       chunks
+                    Should contain      ${result}       files
```

Comment:
Can this command be executed on the datanode container and lets do a check that the file exists.

Reply:
I have included a separate robot test that checks whether the file is present in the datanode.